scalafix-core/shared/src/main/scala/scalafix/rewrite/RewriteCtx.scala

```diff
@@ -30,8 +30,21 @@ case class RewriteCtx(tree: Tree, config: ScalafixConfig) extends PatchOps {
   lazy val tokenList: TokenList = new TokenList(tokens)
   lazy val matching: MatchingParens = MatchingParens(tokens)
   lazy val comments: AssociatedComments = AssociatedComments(tokens)
+  lazy val input: Input = tokens.head.input
   val reporter: ScalafixReporter = config.reporter
 
+  // Debug utilities
+  def mirror(implicit mirror: Mirror): Database =
```

Comment:
Good catch, I'll let it return a Mirror instead. I find the database vs. mirror distinction confusing, ideally we should consolidate them. cc/ @xeno-by